### ISSUE TITLE
Update to Dragon 0.4.0

### ISSUE THREAD

**Issue by oubiwann** [clozhang/blog#8]
This is blocked by https://github.com/clojusc/dragon/issues/28.

**Comment by zivl** [comment]
has anyone taken this already?

**Comment by oubiwann** [comment]
Yeah, I should have updated it. It's a bit involved, too -- there are several features that have been implemented in other projects that need to be brought into Dragon before this one can be updated to use the latest Dragon (thus reducing & preventing further redundancies).
